public/Board.js

```diff
@@ -54,10 +60,24 @@ export default class Board {
 
       currentPlayerBeingDealt.receiveCard(card);
 
-      await sleepNow(1000);
+      await sleepNow(500);
     }
   }
 
+  async sartWithGameDealing() {
```

Comment:
There is a typo here:
```suggestion
  async startWithGameDealing() {
```